paddle/optimizer/Tensor.h

```diff
@@ -0,0 +1,49 @@
+#ifndef PADDLE_OPTIMIZER_TENSOR_H_
```

Comment:
In a formal discussion, we decided to use `#pragma once`(https://github.com/PaddlePaddle/cpp-primer-digest). So we can first change the digest then change our code style.